### PR TITLE
Add support for Scalable Storage

### PR DESCRIPTION
**Pull request by dweinshenker**
Scalable storage was released for MySQL and PostgreSQL:

https://www.digitalocean.com/blog/introducing-digitalocean-scalable-storage 

It is also supported now in `godo` in v1.105.0: https://github.com/digitalocean/godo/releases/tag/v1.105.0

### CODE REVIEW

File: digitalocean/database/resource_database_cluster_test.go [hunk 1]
```diff
@@ -105,6 +107,37 @@ func TestAccDigitalOceanDatabaseCluster_WithUpdate(t *testing.T) {
...
+		Steps: []resource.TestStep{
+			{
+				Config: fmt.Sprintf(testAccCheckDigitalOceanDatabaseClusterConfigBasic, databaseName),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckDigitalOceanDatabaseClusterExists("digitalocean_database_cluster.foobar", &database),
+					testAccCheckDigitalOceanDatabaseClusterAttributes(&database, databaseName),
+					resource.TestCheckResourceAttr(
+						"digitalocean_database_cluster.foobar", "storage_size_mib", "30720"),
+				),
+			},
```
[the remark this answers is not in the document]
Comment (dweinshenker) [reply digitalocean/database/resource_database_cluster_test.go:127]: Ah that's a good suggestion @andrewsomething, I tested this myself 1-2x, but must've missed the race condition in those runs. I'll add that sleep as part of the test

**Review by andrewsomething — APPROVED** [review]
:+1:  LGTM!

**Comment by yordis** [comment]
@andrewsomething I know you just merged, but we need this at the moment, so I am wondering what is the release cadence of the package.

Is there any possibility to cut a release?

**Comment by andrewsomething** [comment]
@yordis Once this PR is merged, we'll be tagging a new release https://github.com/digitalocean/terraform-provider-digitalocean/pull/1055

**Pull request merged**